### DISCUSSION TITLE
MacWhisper - Full featured native Whisper for Mac

### DISCUSSION THREAD

**Discussion by jordibruin** [openai/whisper#1092] [Show and tell]
Hi all, I built MacWhisper recently. A native SwiftUI that runs Whisper locally on your Mac. I've been building it out over the past two months with advanced exports (html, pdf and the basics such as srt), batch transcription, speaker selection, GPT prompting, translation, global find and replace and more. I'm adding system audio transcription as well as podcast transcription with multiple speakers this week.

You can get it for free at www.macwhisper.com or from the Mac App Store https://apps.apple.com/us/app/whisper-transcription/id1668083311?mt=12

Edit in 2026: Three years later and MacWhisper is still in active development and better than ever 🙂 
It supports more models such as Parakeet, can record your meetings, you can dictate in any app (with the free version as well), view live captions, export into a ton of file formats and much more. [This link](https://goodsnooze.gumroad.com/l/macwhisper/mwgit) will give you 10% off a pro license (one time payment).

**Comment by tristan-mcinnis** [comment]
Where is the translation? I've emailed you about this. I paid for the pro version. Thanks.
